lib/cached_counts.rb

```diff
@@ -5,46 +5,62 @@ module CachedCounts
   extend ActiveSupport::Concern
 
   module ClassMethods
-    # Cache the count for a scope in memcached.
+    # Cache the count for a scope in Memcached.
     #
-    # e.g.
-    #   User.caches_count_where :confirmed
-    #   > User.confirmed_count # User.confirmed.count, but cached
+    # For example, if you have a model:
+    #   class Post < ActiveRecord::Base
+    #     scope :sponsored, -> { where(sponsored: true) }
+    #     caches_count_where :sponsored
+    #   end
     #
-    # Automatically adds after_commit hooks which increment/decrement the value
-    # in memcached when needed. Queries the db on cache miss.
+    # Then you can call
+    #   Post.sponsored_count
+    # to fetch the number of sponsored posts from Memcached.
+    #
+    # Defines a few class methods:
+    # [+#{attribute_name}_count+]
+    #   The number of models in the scope.
+    # [+#{attribute_name}_key+]
+    #   The key in Memcached that stores the count.
+    # [+#{attribute_name}_count=(value)+]
+    #   Sets the value of the count.
```

Comment:
i think it might be worth adding parenthetically here that you don't need to call this except for testing